internal/network/command.go

```diff
@@ -15,29 +15,43 @@ import (
-	Id                    string `human:"ID"`
-	EnvironmentId         string `human:"Environment"`
-	Name                  string `human:"Name"`
-	Cloud                 string `human:"Cloud"`
-	Region                string `human:"Region"`
-	Cidr                  string `human:"CIDR"`
-	Zones                 string `human:"Zones"`
-	DnsResolution         string `human:"DNS Resolution"`
-	Phase                 string `human:"Phase"`
-	ActiveConnectionTypes string `human:"Active Connection Types"`
+	Id                       string `human:"ID"`
+	EnvironmentId            string `human:"Environment"`
+	Name                     string `human:"Name"`
+	Cloud                    string `human:"Cloud"`
+	Region                   string `human:"Region"`
+	Cidr                     string `human:"CIDR"`
+	Zones                    string `human:"Zones"`
+	DnsResolution            string `human:"DNS Resolution,omitempty"`
+	Phase                    string `human:"Phase"`
+	SupportedConnectionTypes string `human:"Supported Connection Types"`
+	ActiveConnectionTypes    string `human:"Active Connection Types"`
+	Vpc                      string `human:"VPC"`
+	Account                  string `human:"Account"`
+	Project                  string `human:"Project"`
+	VpcNetwork               string `human:"VPC Network"`
+	VNet                     string `human:"VNet"`
+	Subscription             string `human:"Subscription"`
 }
 
 type serializedOut struct {
-	Id                    string   `serialized:"id"`
-	EnvironmentId         string   `serialized:"environment_id"`
-	Name                  string   `serialized:"name"`
-	Cloud                 string   `serialized:"cloud"`
-	Region                string   `serialized:"region"`
-	Cidr                  string   `serialized:"cidr"`
-	Zones                 []string `serialized:"zones"`
-	DnsResolution         string   `serialized:"dns_resolution"`
-	Phase                 string   `serialized:"phase"`
-	ActiveConnectionTypes []string `serialized:"active_connection_types"`
+	Id                       string   `serialized:"id"`
+	EnvironmentId            string   `serialized:"environment_id"`
+	Name                     string   `serialized:"name"`
+	Cloud                    string   `serialized:"cloud"`
+	Region                   string   `serialized:"region"`
+	Cidr                     string   `serialized:"cidr"`
+	Zones                    []string `serialized:"zones"`
+	DnsResolution            string   `serialized:"dns_resolution"`
+	Phase                    string   `serialized:"phase"`
+	SupportedConnectionTypes []string `serialized:"supported_connection_types"`
+	ActiveConnectionTypes    []string `serialized:"active_connection_types"`
+	Vpc                      string   `serialized:"vpc"`
+	Account                  string   `serialized:"account"`
+	Project                  string   `serialized:"project"`
+	VpcNetwork               string   `serialized:"vpc_network"`
+	VNet                     string   `serialized:"vnet"`
```

Comment:
What does "v" stand for?
```suggestion
	VNet                     string   `serialized:"v_net"`
```

Reply:
Azure's "Virtual Network", I am matching this with our API response here. Do you think that we should keep it consistent with the API response?
![image](https://github.com/confluentinc/cli/assets/135164550/936fb180-d1f9-4d40-9bd5-abb5a2a9cc9e)

Reply:
As someone who knows nothing about this networking feature, I think I'd prefer "Virtual Network" over "VNet"

Reply:
Just checked with our PM @kabraham13 that we think it's fair to assume users of this networking feature should know what a VPC or a VNet is, considering that users can specify which cloud provider to use while creating a Confluent network.

Can we keep it as "VNet"?

Reply:
Sure, if it's a common term!